.golangci.yml

```diff
@@ -34,7 +34,7 @@ linters:
     - revive
     - stylecheck
     - whitespace
-    - wsl
+    # - wsl
```

Comment:
I suggest leaving `wsl` in here since theres other users of other libraries in this repo that would prefer to have it around.